### DISCUSSION TITLE
Group By

### DISCUSSION THREAD

**Discussion by 12re34enes** [apache/superset#27363] [General]
### Bug description

I have a table pattern and I need to make group by from the charts section for this table, but there is no group by section. How can I do something for this, I can easily do it from the query part, but then I cannot filter from columns that are not in the table.

### How to reproduce the bug

Charts Side 
![Group by](https://github.com/apache/superset/assets/155597786/9330cb90-2601-4bed-a5c7-c35f1382076a)

### Superset version

3.1.1

### Python version

3.10

### Node version

16

### Browser

Chrome

### Checklist

- [X] I have searched Superset docs and Slack and didn't find a solution to my problem.
- [X] I have searched the GitHub issue tracker and didn't find a similar bug report.
- [ ] I have checked Superset's logs for errors and if I found a relevant Python stacktrace, I included it here as text in the "additional context" section.

**Comment by michael-s-molina** [comment]
@12re34enes Please follow the issue template recommendation:

🙏 Asking a question or getting help: post in the [Superset Slack chat](http://bit.ly/join-superset-slack) or [GitHub Discussions](https://github.com/apache/superset/discussions) under "Q&A / Help".

I'm converting this to a GitHub discussion as it's more appropriate. It can also be a Slack thread.
